src/components/Card/Card.jsx

```diff
@@ -0,0 +1,21 @@
+const Card = (product) => {
+  const { title, categoryName, brand, imageUrl, price } = product.product;
```

Comment:
Destructure it like  `Card( { title, categoryName } )` or simple use `Card( props )`